### ISSUE TITLE
About two pictures, one normal and one defective test?

### ISSUE THREAD

**Issue by codeman008** [zqhang/AnomalyCLIP#52]
First of all, thank you very much for the project code you provided. I now have two pictures, one normal and one defective. Can I directly use the weight file provided in the project for testing?

**Comment by zqhang** [comment]
Yes, please refer to the [script](https://github.com/zqhang/AnomalyCLIP/blob/main/test_one_example.sh)
